ulid.go

```diff
@@ -41,13 +41,17 @@ An ULID is a 16 byte Universally Unique Lexicographically Sortable Identifier
 type ULID [16]byte
 
 var (
-	// ErrDataSize is returned when parsing or unmarshaling ULIDs with the wrong
+	// ErrDataSize is returned when parsing or unmarshalling ULIDs with the wrong
```

Comment:
`unmarshaling` is the preferred spelling.

Reply:
Ah, no problem. Fixed.